man/mono.1

```diff
@@ -348,6 +403,12 @@ will switch to '/bin/mono64' iff '/bin/mono' is a 32-bit build).
 \fB--help\fR, \fB-h\fR
 Displays usage instructions.
 .TP
+\fB--hybrid-aot\fR
+This is an experimental flag that instructs the Mono runtime to not
+generate any IL at runtime and depend exclusively on the code
```

Comment:
This is wrong.

Under hybrid-aot the following conditions apply:

- JIT compilation is enabled;
- dynamic codegen APIs, such SRE, are available;
- dynamic assembly loading is available;
- AOT images must have being compiled in the hybrid-aot mode.
- the runtime will not use IL from assemblies with AOT images (implying they can be stripped)